views/index.erb

```diff
@@ -1,9 +1,9 @@
-<h1>All My Students</h1>
+<h1>All My Favorite Bands</h1>
 <ul>
-  <% students.each do |student| %>
+  <% bands.each do |band| %>
     <li>
-      <a href="mailto:<%= student[:email] %>"><%= student[:name] %></a>
-      (<%= student[:discord] %>)
+      <a href="mailto:<%= band[:firstalbumyear] %>"><%= student[:name] %></a>
+      (<%= student[:myrank] %>)
```

Comment:
I don't think you successfully ran your app locally. 😄 

Here again, and on the line above, Ruby would have complained about `undefined variable 'student'`. You renamed the block argument from `|student|` to `|band|`, but you didn't change all the spots inside the block that referenced `student`! A simple find+replace in your editor would have helped you out here.